setup/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "setup-notation",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
It is better to have a `build` script for quicker build just like a `Makefile`.

Reply:
build has been added to the package.json